contracts/utils/SafeCast.sol

```diff
@@ -94,4 +94,17 @@ library SafeCast {
         require(value < 2**8, "SafeCast: value doesn\'t fit in 8 bits");
         return uint8(value);
     }
+
+    /**
+     * @dev Returns the unsigned uint256 from int256, reverting if
+     * the input is negative.
+     *
+     * Requirements:
+     *
+     * - input must be greater or equal than 0.
+     */
+    function toUint256(int256 value) internal pure returns (uint256) {
+        require(value >= 0, "SafeCast: negative value cannot be cast to uint256");
```

Comment:
```suggestion
        require(value >= 0, "SafeCast: value must be positive");
```

We try to explain what went wrong in these, instead of attempting to go over restrictions or explain what should've happened.